spynnaker/pyNN/models/neural_projections/connectors/from_list_connector.py

```diff
@@ -262,8 +157,3 @@ def conn_list(self):
     @conn_list.setter
     def conn_list(self, new_value):
         self._conn_list = new_value
```

Comment:
This is quite similar to what I tried to do in https://github.com/SpiNNakerManchester/sPyNNaker/pull/507 - perhaps that PR isn't necessary any more.